app/recipe/[slug]/recipe.interface.ts

```diff
@@ -0,0 +1,16 @@
+interface ImageProps {
+  title: string;
+  url: string;
+  width: number;
+  height: number;
+}
+
+export interface RecipeProps {
+  title: string;
+  slug: string;
+  cookingTime: string;
+  ingredients: string[];
+  method: any;
```

Comment:
Preferably, we should always define prop types and avoid using any as the type. Contentful RTE field data can be defined as follows:

```
export interface RichtextProps {
  json: Document;
  links?: {
    assets: {
      hyperlink?: {
        sys: {
          id: string;
        };
        url: string;
      }[];
      block: {
        sys: {
          id: string;
        };
        contentType?: string;
        url: string;
        title: string;
        description: string;
        width: number;
        height: number;
      }[];
    };
  };
}
```